youtube_dl/extractor/digitalconcerthall.py

```diff
@@ -0,0 +1,110 @@
+        playlist_title = self._html_search_regex(r'<title>(.+?)</title>', webpage, 'title') or \
+            self._og_search_title(webpage)
+        self.debug_out("playlist_title: " + playlist_title)
+
+        # this returns JSON containing the urls of the playlist
+        # Note:  you must be authenticated to get the stream info
+        playlist_dict = self._download_json(
+            'https://www.digitalconcerthall.com/json_services/get_stream_urls?id='
+            + video_id + "&language=" + language, video_id, note='Downloading Stream JSON').get('urls')
+        # use the API to get other information about the concert
+        vid_info_dict = self._download_json(
+            'https://api.digitalconcerthall.com/v2/concert/'
+            + video_id, video_id, headers={'Accept': 'application/json',
+                                           'Accept-Language': language})
+        embedded = vid_info_dict.get('_embedded')
+
+        entries = []
+        for key in playlist_dict:
+            self.debug_out("key: " + key)
+            m3u8_url = playlist_dict.get(key)[0].get('url')
+            self.debug_out("key url: " + m3u8_url)
+            formats = self._extract_m3u8_formats(
+                m3u8_url, key, 'mp4', 'm3u8_native', m3u8_id='hls', fatal=False)
+            self.debug_out(formats)
+            flat_list = []
+            for embed_type in embedded:
+                for item in embedded.get(embed_type):
+                    if embed_type == 'interview':
+                        item['is_interview'] = 1
+                    else:
+                        item['is_interview'] = 0
+                    flat_list.append(item)
+            vid_info = [x for x in flat_list if x.get('id') == key][0]
+            if vid_info.get('is_interview') == 1:
+                title = "Interview - " + vid_info.get('title', "unknown interview title")
+            else:
+                title = (vid_info.get('name_composer') if vid_info.get('name_composer')
+                         else 'unknown composer') + ' - ' + vid_info.get('title', "unknown title")
+            duration = vid_info.get('duration_total')
+            # avoid filenames that exceed filesystem limits
+            title = (title[:MAX_TITLE_LENGTH] + '..') if len(title) > MAX_TITLE_LENGTH else title
+            # append the duration in minutes to the title
+            title = title + " (" + str(round(duration / 60)) + " min.)"
+            self.debug_out("title: " + title)
+            timestamp = vid_info.get('date').get('published')
+            entries.append({
+                'id': key,
+                'title': title,
+                'url': m3u8_url,
+                'formats': formats,
+                'duration': duration,
+                'timestamp': timestamp,
+            })
+            if vid_info_dict.get('short_description'):
+                entries[-1]['description'] = vid_info_dict.get('short_description')
```

Comment:
Adding the playlist description to each video was deliberate, because it looks like only interviews have their own short_description attributes.  So I need to fix it to add the interview description, if it exists.

Reply:
Resolved with https://github.com/teridon/youtube-dl/commit/09014a88ca55fc26cdd342c7f61225b346c1a0c4